app/api/v2/admin/jobs.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+module API
+  module V2
+    module Admin
+      # Admin functionality over jobs table
+      class Jobs < Grape::API
+        resource :jobs do
+          helpers ::API::V2::NamedParams
+
+          desc 'Create new job',
+            failure: [
+              { code: 401, message: 'Invalid bearer token' }
+            ],
+            success: { code: 200, message: 'Job was created' }
+          params do
+            requires :description,
+                     type: String,
+                     allow_blank: false,
+                     desc: 'job description'
+            requires :type,
+                     type: String,
+                     values: { value: -> { Job::TYPES }, message: 'admin.job.invalid_type'},
+                     desc: 'job type'
+            requires :start_at,
```

Comment:
can be optional

Reply:
How to schedule the worker If `start_at` is nil?

Do you mean to have logic like this? And How about `finish_at`?

```ruby
if start_at.nil?
  Worker.perform_async
else
  Worker.perform_at(start_at)
end
```

Reply:
oh, sorry, finish_at can optional, you can schedule maintenance without end date and whitelist your IP to disable maintenance from Tower UI

Reply:
Changed.